sigopt/factory.py

```diff
@@ -100,46 +99,52 @@ def upload_runs(self, runs, max_batch_size=10000):
       )
     return result
 
-  def create_prevalidated_experiment(self, validated_body):
+  def create_prevalidated_aiexperiment(self, validated_body):
     connection = self.connection
     client_id, project_id = self.ensure_project_exists()
-    experiment = connection.clients(client_id).experiments().create(
-      **{PROJECT_KEY: project_id, RUNS_ONLY_KEY: True},
+    aiexperiment = connection.clients(client_id).projects(project_id).aiexperiments().create(
       **validated_body,
     )
-    self._on_experiment_created(experiment)
-    return ExperimentContext(experiment, connection=connection)
+    self._on_aiexperiment_created(aiexperiment)
+    return AIExperimentContext(aiexperiment, connection=connection)
 
-  def create_experiment(self, *, name, parameters, metrics, **experiment_body):
+  def create_experiment(self, *args, **kwargs):
+    return self.create_aiexperiment(*args, **kwargs)
+
+  def create_aiexperiment(self, *, name, parameters, metrics, **aiexperiment_body):
     # name, parameters and metrics are always required and placing them in the signature
     # results in more pythonic errors
-    experiment_body["name"] = name
-    experiment_body["parameters"] = parameters
-    experiment_body["metrics"] = metrics
-    validated = validate_experiment_input(experiment_body)
-    return self.create_prevalidated_experiment(validated)
+    aiexperiment_body["name"] = name
+    aiexperiment_body["parameters"] = parameters
+    aiexperiment_body["metrics"] = metrics
+    validated = validate_aiexperiment_input(aiexperiment_body)
+    return self.create_prevalidated_aiexperiment(validated)
 
   def get_experiment(self, experiment_id):
```

Comment:
maintaining this function? or should we dispense with it completely in favor of `get_aiexperiment()`?

Reply:
Maintaining it for now b/c existing documentation and user code references it.